### ISSUE TITLE
Error occured while querying the personal wallet journal.

### ISSUE THREAD

**Issue by rmcginnis** [evemondevteam/evemon#16]
When I click API error  details please see attached file for the full stack trace. 
[new 1.txt](https://github.com/evemondevteam/evemon/files/402461/new.1.txt)

**Comment by Crowles** [comment]
Hi @rmcginnis, Just to make you aware you've attached the XML containing your personal wallet data, not the stack trace, the stack trace can be found in /%APPDATA%/EVEMon.

+1 I'm also encountering this issue, stack trace:

```xml
0d 0h 10m 16s > ExceptionHandler.LogException - Handled exception
    System.InvalidOperationException: There is an error in XML document (41, 189). ---> System.OverflowException: Value was either too large or too small for an Int32.
   at System.Number.ParseInt32(String s, NumberStyles style, NumberFormatInfo info)
   at Microsoft.Xml.Serialization.GeneratedAssembly.XmlSerializationReaderCCPAPIResult1.Read3_Item(Boolean isNullable, Boolean checkType)
   at Microsoft.Xml.Serialization.GeneratedAssembly.XmlSerializationReaderCCPAPIResult1.Read4_SerializableAPIWalletJournal(Boolean isNullable, Boolean checkType)
   at Microsoft.Xml.Serialization.GeneratedAssembly.XmlSerializationReaderCCPAPIResult1.Read5_Item(Boolean isNullable, Boolean checkType)
   at Microsoft.Xml.Serialization.GeneratedAssembly.XmlSerializationReaderCCPAPIResult1.Read6_eveapi()
   --- End of inner exception stack trace ---
   at System.Xml.Serialization.XmlSerializer.Deserialize(XmlReader xmlReader, String encodingStyle, XmlDeserializationEvents events)
   at System.Xml.Serialization.XmlSerializer.Deserialize(Stream stream)
   at EVEMon.Common.Util.DeserializeAPIResultCore[T](IXPathNavigable doc, XslCompiledTransform transform) in EVEMon.Common\Util.cs:line 386
```

I'm running version 3.0.2, downloaded today from https://evemondevteam.github.io/evemon/.

**Comment by Endeavourl** [comment]
This is also happening for me since a couple of days ago.

The error is `XML error: There is an error in XML document (54, 185)`.
The 54th xml line is
```xml
      <row date="2016-10-xx xx:xx:xx" refID="131073xxxxx" refTypeID="10" ownerName1="xxxxxxxxxxxx" ownerID1="9117xxxx" ownerName2="xxxxxxxxxx" ownerID2="19452xxxxx" argName1="" argID1="0" amount="xxxxxx.00" balance="xxxxxxxxxx.80" reason="DESC: &quot;\u0414\u0440\u043E\u043D\u044B)&quot;&#xA;" taxReceiverID="" taxAmount="" owner1TypeID="1375" owner2TypeID="1374" />
```
and the error is at
```
argID1="0"
       ^
```
which looks very ordinary, in fact lines next to this one have attributes like this one.

Stacktrace:
```
0d 0h 20m 14s > ExceptionHandler.LogException - Handled exception
    System.InvalidOperationException: There is an error in XML document (54, 185). ---> System.OverflowException: Value was either too large or too small for an Int32.
   at System.Number.ParseInt32(String s, NumberStyles style, NumberFormatInfo info)
   at Microsoft.Xml.Serialization.GeneratedAssembly.XmlSerializationReaderCCPAPIResult1.Read3_Item(Boolean isNullable, Boolean checkType)
   at Microsoft.Xml.Serialization.GeneratedAssembly.XmlSerializationReaderCCPAPIResult1.Read4_SerializableAPIWalletJournal(Boolean isNullable, Boolean checkType)
   at Microsoft.Xml.Serialization.GeneratedAssembly.XmlSerializationReaderCCPAPIResult1.Read5_Item(Boolean isNullable, Boolean checkType)
   at Microsoft.Xml.Serialization.GeneratedAssembly.XmlSerializationReaderCCPAPIResult1.Read6_eveapi()
   --- End of inner exception stack trace ---
   at System.Xml.Serialization.XmlSerializer.Deserialize(XmlReader xmlReader, String encodingStyle, XmlDeserializationEvents events)
   at System.Xml.Serialization.XmlSerializer.Deserialize(Stream stream)
   at EVEMon.Common.Util.DeserializeAPIResultCore[T](IXPathNavigable doc, XslCompiledTransform transform) in EVEMon.Common\Util.cs:line 386
```

**Comment by elliena-bulmer** [comment]
@rmcginnis & @Crowles , Should be fixed with #23, which should be in the next release. If possible, I'd encourage you to build the current code to see if it's fixed on your side.

**Comment by Crowles** [comment]
@elliena-bulmer just compiled and tested on my side, can confirm the issue has been fixed, wallet journal and wallet transactions are both displaying all data, including from citadels. Thanks!

![image](https://cloud.githubusercontent.com/assets/6986645/20813129/45e7a58a-b80c-11e6-98d4-6db8999e7210.png)

***
![image](https://cloud.githubusercontent.com/assets/6986645/20813153/5f4b0ecc-b80c-11e6-95b3-be289c6daff2.png)

**Comment by Ashilta** [comment]
This has been released as part of v3.0.3!
